src/main/java/com/example/fooddelivery/controller/CreateOrderController.java

```diff
@@ -25,25 +29,56 @@
     private static final String ADDRESS_FORM = "addressForm";
     private static final String RESTAURANTS = "restaurants";
     private static final String ADDED_PRODUCTS = "addedProducts";
+    private static final String ORDER_ID = "orderId";
+    private static final String ORDERS_LIST = "ordersList";
+    private static final String ORDER_DETAILS = "orderDetails";
     private static final String SESSION_PRODUCTS = "sessionProducts";
     private static final String SESSION_ADDRESS = "sessionAddress";
     private static final String REDIRECT = "redirect:/";
 
     @Autowired
-    private DefaultRestaurantService defaultRestaurantService;
+    private RestaurantService restaurantService;
 
     @Autowired
-    private DefaultCartFacade defaultCartFacade;
+    private CartFacade cartFacade;
+
+    @Autowired
+    private OrderService orderService;
+
+    @Autowired
+    private OrderFacade orderFacade;
 
     @GetMapping()
     public String createOrder(Model model) {
         model.addAttribute(ADDRESS_FORM, new AddressForm());
         model.addAttribute(ADDED_PRODUCTS, new ArrayList<ProductWithQuantityDto>());
-        model.addAttribute(RESTAURANTS, defaultRestaurantService.getRestaurantsNameAndId());
+        model.addAttribute(RESTAURANTS, restaurantService.getRestaurantsNameAndId());
 
         return CREATE_ORDER_PAGE;
     }
 
+    @GetMapping("/details/{orderId}")
+    public String getOrder(@PathVariable("orderId") String orderId, Model model) {
+        try {
+            model.addAttribute(ORDER_DETAILS, orderFacade.getOrderDetails(Long.parseLong(orderId)));
+            model.addAttribute(ORDER_ID, orderId);
+            return ORDER_DETAILS_PAGE;
+        } catch (Exception e) {
+            return ERROR_PAGE;
+        }
+    }
+
+    @GetMapping("/list")
+    public String getOrdersList(Model model) {
+        model.addAttribute(ORDERS_LIST, orderService.getOpenOrders());
```

Comment:
Please use only facade in controller

Reply:
I'm not sure about this approach. Please give me a link why you think is better to move in a facade if don't have any logic which involves multiple services for example.